xml/System.Text.Json/JsonSerializer.xml

```diff
@@ -306,8 +306,11 @@ For more information, see [How to serialize and deserialize JSON](/dotnet/standa
 
 ]]></format>
         </remarks>
+        <exception cref="T:System.IndexOutOfRangeException">
+          .NET Core 3.0 and earlier versions only: <typeparamref name="TValue" /> is <see cref="T:System.Char" /> and <paramref name="json" /> is an empty string.</exception>
         <exception cref="T:System.ArgumentNullException">
-          <paramref name="json" /> is <see langword="null" />.</exception>
+          <paramref name="json" /> is <see langword="null" /> (for example, <code>JsonSerializer.Deserialize&lt;char&gt;((string)null);
+</code>).</exception>
```

Comment:
Somehow this row is not rendered correctly, anyway not sure if we need this example here as the exception apply to any type parameter not only  `char`